src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -38,6 +38,8 @@ class Definition
     private $public = true;
     private $synthetic = false;
     private $abstract = false;
+    private $deprecated = false;
+    private $deprecatedMessage;
```

Comment:
Yes, first I went for a separate attribute (this one is the left-over), but then I chose to go for an array. I thought it was cleaner to have only one attribute, but then again, I can split it